src/components/o-modal/_template.js

```diff
@@ -0,0 +1,23 @@
+import html from 'nanohtml';
+
+export default ({ open }, childrenFragment) => {
+  const closeModal = () => {
+    const modal = document.getElementById('modal-wrapper');
+    modal.style.display = 'none';
```

Comment:
please use Modifier class instead of overriding style.

Reason: Specificity increases in this way too much which can lead to difficulty in overriding styles